src/libraries/System.Net.Security/tests/UnitTests/SslStreamCertificateContextOcspLinuxTests.cs

```diff
@@ -18,6 +18,7 @@ namespace System.Net.Security.Tests;
 public class SslStreamCertificateContextOcspLinuxTests
 {
     [Fact]
+    [PlatformSpecific(TestPlatforms.Linux)]
```

Comment:
The class is named "OcspLinuxTests", can/should this attribute just be put on the class? (I think it works at that level)